src/shmessy/types/utils.py

```diff
@@ -0,0 +1,19 @@
+import logging
+from datetime import datetime
+
+from numpy import ndarray
+
+logger = logging.getLogger(__name__)
+
```

Comment:
@DanielAvdar I am loading the types dynamically. Take a look at types_handler.discover_types.
I have a feeling that this new file will be recognized as type and failed because it's not a type implementation